src/assets/htmx.md

```diff
@@ -0,0 +1,347 @@
+---
+title: htmx
+authorName: Niraj Acharya
+authorAvatar: https://avatars.githubusercontent.com/u/122071597?v=4
+authorLink: https://github.com/nirajacharya2
+createdAt: Sep 29, 2023
+tags: js, html, go, htmx
+banner: https://raw.githubusercontent.com/JankariTech/blog/master/src/imgs/fallback_banner.png
```

Comment:
maybe replace this with something more specific relating to your blog post?

Reply:
that should be fine. you could add the link to this github repo and the official documentation at the end of the blog as declaration of source and for further reference for the readers.